docs/pages/application-access/cloud-apis/google-cloud.mdx

```diff
@@ -0,0 +1,615 @@
+Google Cloud.
+
+### Run the Teleport Application Service
+
+On the host where you will run the Teleport Application Service, execute the
+following command, depending on whether you installed Teleport using a package
+manager or via a TAR archive:
+
+<Tabs>
+<TabItem label="Package Manager">
+
+```code
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem label="TAR archive">
+
+```code
+$ sudo teleport install systemd --output=/etc/systemd/system/teleport.service;
+$ sudo systemctl enable teleport; 
+$ sudo systemctl start teleport; 
+```
+</TabItem>
+</Tabs>
+
+## Step 3/4. Enable your user to access Google Cloud CLIs
+
+The next step is to authorize your Teleport user to access a Google Cloud
+service account and execute Google Cloud CLI commands via Teleport. You will
+protect access to the service account using Teleport's RBAC system, where a
+user's roles determine which Google Cloud service accounts (if any) they can
+access.
+
+There are two approaches you can take to authorize users to access Google Cloud
+service accounts:
+
+|Approach|Description|Supported User Types|
+|---|---|---|
+|**Dynamic**|A Teleport role includes a template variable that grants a user access to all Google Cloud service accounts assigned directly to them.|Local users, OIDC, SAML|
+|**Static**|A Teleport role explicitly specifies the Google Cloud service accounts a user is allowed to assume.|Local users, OIDC, SAML, GitHub|
+
+<Notice type="tip">
+
+We recommend using the dynamic approach, since it scales more easily as you add
+service accounts to your Google Cloud account. If you have configured an open
+source Teleport cluster to authenticate users via GitHub SSO, you must use the
+static approach, as OAuth-based GitHub applications do not support custom
+claims.
+
+</Notice>
+
+<Tabs dropdownCaption="Approach">
+<TabItem options="Dynamic Identities" label="Local Users">
+
+Create a file called `google-cloud-cli-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: google-cloud-cli-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    gcp_service_accounts:
+      - '{{internal.gcp_service_accounts}}'
+```
+
+When a user with the `google-cloud-cli-access` role authenticates to a Google
+Cloud CLI via Teleport, the Teleport Auth Service populates the
+`{{internal.gcp_service_accounts}}` template variable with any Google Cloud
+service accounts you have assigned to the user.
+
+Assign the `teleport-vm-lister` service account we created earlier (or another
+service account) to your Teleport user by running the following command:
+
+```code
+$ tctl users update <Var name="teleport-user" /> \
+--set-gcp-service-accounts teleport-vm-lister@${PROJECT_ID?}.iam.gserviceaccount.com
+```
+
+This command uses the `--set-gcp-service-accounts` flag to add Google Cloud
+service accounts to a user. You can assign `--set-gcp-service-accounts` to
+multiple service account URIs, separated by commas.
```

Comment:
```suggestion
service accounts to a user. You can use `--set-gcp-service-accounts` to
assign multiple service account URIs, separated by commas.
```

Reply:
I've used a variation of this that makes it clearer that the comma-separated list is a value of `--set-gcp-service-accounts`